tests/src/Kernel/EncryptStreamTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Drupal\Tests\file_encrypt\Kernel;
+
+use Drupal\file_encrypt\EncryptStreamWrapper;
+
+/**
+ * @group file_encrypt
+ */
+class EncryptStreamTest extends FileEncryptTestBase {
```

Comment:
no, i see "testDecrypt", and was curious if there should be a "testEncrypt" as well